app/ore/project/io/ProjectFiles.scala

```diff
@@ -41,14 +51,15 @@ class ProjectFiles(val env: OreEnv) {
     * @param newName  New project name
     * @return         New path
     */
-  def renameProject(owner: String, oldName: String, newName: String): Try[Unit] = Try {
+  def renameProject(owner: String, oldName: String, newName: String): Try[Unit] = Try { //TODO
```

Comment:
TODO what?

Reply:
Fixed it in latest commit and forgot to remove this. Will fix this up tomorrow.